index.php

```diff
@@ -9,8 +9,9 @@
 }
 
 $context['description'] = get_the_archive_description('<div class="archive-description">', '</div>');
+
 $context['pagination'] = Timber::get_pagination([
-    'mid_size' => 2,
+    'mid_size' => 1,
```

Comment:
I personally also preferred a `mid_size` of 2 (in most cases this looks and works better, because there are often 6-7 pages that don't need to be split up necessarily), but that's just my opinion.